api/routes/tag_routes.py

```diff
@@ -60,6 +72,30 @@ def add_tag_for_fund_round(fund_id, round_id):
     abort(404)
 
 
+def update_tags_for_fund_round(fund_id, round_id):
+    args = request.get_json()
+
+    tags = [
+        {
+            "id": arg.get("id"),
+            "value": arg.get("value"),
+            "tag_type_id": arg.get("tag_type_id"),
+            "creator_user_id": arg.get("creator_user_id"),
+            "active": arg.get("active"),
+        }
+        for arg in args
+    ]
+
+    tags = update_tags(tags, fund_id, round_id)
+
+    if tags:
+        serialiser = TagSchema()
+        serialised_tags = [serialiser.dump(r) for r in tags]
+        return serialised_tags
+
+    abort(404)
```

Comment:
Might be nice to log if this ever happens, probably mean a tag hasn't been updated for whatever reason